lib/repo_tools.pm

```diff
@@ -52,9 +53,13 @@ sub add_qa_head_repo {
     add_qa_web_repo();
 
 Helper to add QA web repository repository.
+This repository is *not* mandatory.
 =cut
 sub add_qa_web_repo {
-    zypper_ar(get_required_var('QA_WEB_REPO'), name => 'qa-web', no_gpg_check => 1);
+    my $repo = get_var('QA_WEB_REPO', '');
+    if ($repo) {
```

Comment:
could be one liner ^^ & get_var don't need empty default value

Reply:
@dzedro thx, fixed (didn't notice that `undef` will not be used as string as guarded by `if`. one liner: I thought it'd be more readable, but changed).